### PR TITLE
Add progress bar and progress bar stepper

### PR DESCRIPTION
**Pull request by fredrikloberg**
Added progress bar and progress bar stepper

Screenshot 2021-11-17 at 16 38 38
Screenshot 2021-11-17 at 16 38 50
Screenshot 2021-11-17 at 16 39 19

### CODE REVIEW

**Comment by mathilwa** [comment]
Could this be a part of the already existing stepper? and have two views?

**Comment by fshauge** [comment]
I agree with Mathilde. If we could use media queries to display one or the other or just have an option to choose which one we want.

**Comment by fredrikloberg** [comment]
> Could this be a part of the already existing stepper? and have two views?

Certainly could, for example like we do with buttons. For PartnerHub, we use a custom step indicator for desktop, and the ProgressBarStepper for mobile, so combining them wouldn't yield any benefits for our use.

One thing to note though, is that this ProgressBarStepper and the StepIndicator already in the library work slightly differently. It has one extra parameter, and it's not just a different css styling like it is with button, but a whole new view. Do you think putting them together would make it harder to maintain somehow? 🤔

File: src/Nordea/Components/ProgressBarStepper.elm [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                        Text.textTiny
+                            |> Text.view
+                                [ css [ color Colors.grayDark ] ]
+                                [ Html.text (nextLabel strings.next ++ step) ]
+                    )
+            ]
+        ]
+
+
+strings =
```
Comment (Eivinddh) [inline src/Nordea/Components/ProgressBarStepper.elm:78]: Maybe consider calling it l18n or translation? ☺️  
```suggestion
l18n =
```

Comment (fredrikloberg) [reply src/Nordea/Components/ProgressBarStepper.elm:78]: We have used `strings` for this throughout in PartnerHub, which is also the same name used for this in native Android development. `strings` is already used in the component library as well.

**Comment by fredrikloberg** [comment]
I will merge this now, but feel free to refactor if you think the changes it improve it 🙂

**Pull request merged**